### ISSUE TITLE
Installation fails on MRI 2.3.0

### ISSUE THREAD

**Issue by yob** [robinst/taglib-ruby#67]
When bundling our rails app on MRI 2.3.0, taglib-ruby fails to install. Here's the console output:

```
Installing taglib-ruby 0.7.0 with native extensions

Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /home/jh/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/taglib-ruby-0.7.0/ext/taglib_base
/home/jh/.rbenv/versions/2.3.0/bin/ruby -r ./siteconf20151226-29102-56bry.rb extconf.rb
checking for main() in -lstdc++... yes
checking for main() in -ltag... yes
creating Makefile

To see why this extension failed to compile, please check the mkmf.log which can be found here:

  /home/jh/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/extensions/x86_64-linux/2.3.0-static/taglib-ruby-0.7.0/mkmf.log

current directory: /home/jh/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/taglib-ruby-0.7.0/ext/taglib_base
make "DESTDIR=" clean

current directory: /home/jh/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/taglib-ruby-0.7.0/ext/taglib_base
make "DESTDIR="
compiling taglib_base_wrap.cxx
cc1plus: warning: command line option ‘-Wdeclaration-after-statement’ is valid for C/ObjC but not for C++
cc1plus: warning: command line option ‘-Wimplicit-function-declaration’ is valid for C/ObjC but not for C++
taglib_base_wrap.cxx: In function ‘void SWIG_Ruby_define_class(swig_type_info*)’:
taglib_base_wrap.cxx:1509:9: warning: variable ‘klass’ set but not used [-Wunused-but-set-variable]
   VALUE klass;
         ^
In file included from /home/jh/.rbenv/versions/2.3.0/include/ruby-2.3.0/ruby.h:33:0,
                 from taglib_base_wrap.cxx:848:
taglib_base_wrap.cxx: In function ‘TagLib::StringList ruby_array_to_taglib_string_list(VALUE)’:
/home/jh/.rbenv/versions/2.3.0/include/ruby-2.3.0/ruby/ruby.h:882:52: error: ‘__builtin_choose_expr’ was not declared in this scope
      OBJ_WB_UNPROTECT((VALUE)(obj)), ((VALUE)(obj))))
                                                    ^
/home/jh/.rbenv/versions/2.3.0/include/ruby-2.3.0/ruby/ruby.h:1039:48: note: in definition of macro ‘RARRAY_CONST_PTR’
 #define RARRAY_CONST_PTR(a) rb_array_const_ptr(a)
                                                ^
/home/jh/.rbenv/versions/2.3.0/include/ruby-2.3.0/ruby/ruby.h:1059:50: note: in expansion of macro ‘RB_OBJ_WB_UNPROTECT_FOR’
 #define RARRAY_PTR(a) ((VALUE *)RARRAY_CONST_PTR(RB_OBJ_WB_UNPROTECT_FOR(ARRAY, a)))
                                                  ^
taglib_base_wrap.cxx:1917:15: note: in expansion of macro ‘RARRAY_PTR’
     VALUE e = RARRAY_PTR(ary)[i];
               ^
Makefile:222: recipe for target 'taglib_base_wrap.o' failed
make: *** [taglib_base_wrap.o] Error 1

make failed, exit code 2

Gem files will remain installed in /home/jh/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/taglib-ruby-0.7.0 for inspection.
Results logged to /home/jh/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/extensions/x86_64-linux/2.3.0-static/taglib-ruby-0.7.0/gem_make.out
```

**Comment by robinst** [comment]
Thanks for reporting, created a PR with the fix: #68

**Comment by yob** [comment]
Thanks!

**Comment by robinst** [comment]
Released 0.7.1 with the fix, please try that :).
